### DISCUSSION TITLE
ModuleNotFoundError: No module named 'utils'

### DISCUSSION THREAD

**Discussion by HripsimeS** [mindee/doctr#918] [Q&A] [answered]
Dear Doctr team, 
I am trying your script for the training of text recognition. All installations done correctly following the instruction and did not get an error while importing all libraries/packages mentioned in your script: https://github.com/mindee/doctr/blob/main/references/recognition/train_tensorflow.py

On command prompt going to the folder where my train script is located and then running the model.

```shell
cd C:\Users\Name\references\recognition
python train_tensorflow.py model=sar_resnet31 train_path="C:\Users\Name\Data\train" val_path="C:\Users\Name\Data\val" --vocab legacy_french --pretrained --wb --epochs 50
```

In the end I received the following error 
```
ModuleNotFoundError: No module named 'utils'
```

There  is an error in "from utils import plot_recorder, plot_samples", can't find the module 'utils'.
Do you have an idea why I am receiving this error and how to fix it? Look forward to hearing from you!

**Comment by HripsimeS** [comment, marked as answer]
The issue is fixed :)

**Reply by frgfm** [reply]
Hello @HripsimeS 👋 

Glad you managed to fix your problem! For the sake of the next people looking for a solution, would you mind sharing how you solved this? 🙏

**Reply by HripsimeS** [reply]
Helllo @frgfm. I did installations using a "basic installation", but then I found out it does not provide a deep learning framework which is required for the package to run scripts. Therefore, I installed doctr using a Developer mode.

git clone https://github.com/mindee/doctr.git
pip install -e doctr/.
pip install -e doctr/.[tf]
pip install -e doctr/.[torch]
